staticjinja/reloader.py

```diff
@@ -12,6 +14,9 @@ class Reloader(object):
     """
     def __init__(self, site):
         self.site = site
+        # The following could be part of the Site.__init__ but it would waste
+        # time if the reloader is not used
+        self.site.dep_graph = DepGraph(site)
```

Comment:
I didn't like this either. The only way I could mitigate this was by adding ``self.dep_graph = None`` in ``Site.__init__`` to at least warn readers that this could exist.

If we replace it by ``self.dep_graph`` then we must somehow also move the ``get_dependencies`` method from ``Site`` to ``Reloader``, don't we?